### DISCUSSION TITLE
post/naudio-and-nuget-and-x64-problem

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#85] [Announcements]
# post/naudio-and-nuget-and-x64-problem

Mark Heath's Development Blog

https://markheath.net/post/naudio-and-nuget-and-x64-problem

**Comment by AngelsDragon** [comment]
Hi Mark,
Your courses on Pluralsight has been marked as expired. Would it be possible to place them on Udemy instead?

**Reply by markheath** [reply]
Sadly that is not possible, they still belong to Pluralsight

**Comment by AngelsDragon** [comment]
Hi Mark, 
One more request. Is there a direct way to communicate with you?

**Comment by AngelsDragon** [comment]
Sorry I meant your courses were set to "Retired" not expired.

**Comment by AngelsDragon** [comment]
Thanks Mark. Appreciate it.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
